package.json

```diff
@@ -26,7 +26,7 @@
   ],
   "homepage": "http://brunch.io/",
   "author": "Brunch team (http://brunch.io)",
-  "repository": "https://github.com/brunch/brunch.git",
```

Comment:
I think there may be some parsers / aggregators of NPM package.json files that won't understand the shortcut. Could we change it back?